### PR TITLE
feat:Implemented Even number

### PR DESCRIPTION
**Pull request by Padiprasannakumar123**

### CODE REVIEW

File: programs.js [hunk 1]
```diff
@@ -1 +1,19 @@
-//this file contains logical programs ragarding numbers....
\ No newline at end of file
+//this file contains logical programs ragarding numbers....
+function isEven(number) {
```
Comment (SaiVasudha) [inline programs.js:8]: add documentation for the function

File: programs.js [hunk 2]
```diff
@@ -1 +1,25 @@
...
+ * 
+ * @param {num1} number 
+ * @param {num2} number
+ * @returns return even number
+ */
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+let num1 = 6;
```
Comment (SaiVasudha) [inline programs.js]: instead of num1 and num2  use a and b

File: programs.js
Comment (SaiVasudha) [inline programs.js:1]: correct the commit sentense

Comment (SaiVasudha) [inline programs.js:1]: squash all the commits into single commit

Comment (Padiprasannakumar123) [reply programs.js:1]: I have squashed all the commits

**Review by SaiVasudha — APPROVED** [review]
LGFM

**Pull request merged**